src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const arr = [];
+let total = 0;
+
+for (const text of [...population]) {
+  const temp = +text.innerText.split(',').join('');
+
+  arr.push(temp);
+  total += temp;
+}
+
+const average = total / population.length;
+
+document.querySelector('.total-population')
```

Comment:
I suggest to create a separate variables for such values you get with querySelector